docs/readthedocs/source/doc/Nano/QuickStart/pytorch_quantization_inc.md

```diff
@@ -0,0 +1,62 @@
+# BigDL-Nano PyTorch Quantization with INC Quickstart
+
+**In this guide we will describe how to obtain a quantized model with the APIs delivered by BigDL-Nano in 2 simple steps**
+
+### **Step 0: Prepare Environment**
+We recommend using [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) to prepare the environment. Please refer to the [install guide](../../UserGuide/python.md) for more details.
+
+```bash
+conda create py37 python==3.7.10 setuptools==58.0.4
+conda activate py37
+# nightly bulit version
+pip install --pre --upgrade bigdl-nano[pytorch]
+# set env variables for your conda environment
+source bigdl-nano-init
+```
+
+By default, Intel Neural Compressor is not installed with BigDL-Nano. So if you determine to use it as your quantization backend, you'll need to install it first:
+```bash
+pip install neural-compressor==1.11
+```
+
+### **Step 1: Prepare the Model**
+```python
+import torch
+from torchvision import datasets
+from torch.utils.data.dataloader import DataLoader
+from bigdl.nano.pytorch.vision import transforms
+from bigdl.nano.pytorch.trainer import Trainer
+from torchvision.models import resnet18
+# define your own model
+model = resnet18(num_classes=10)
+loss = torch.nn.CrossEntropyLoss()
+optimizer = torch.optim.Adam(model.parameters(), lr=1e-4)
+model = Trainer.compile(model, loss, optimizer)
+# prepare your own data
+data_transform = transforms.Compose([
+    transforms.Resize((224, 224)),
+    transforms.Grayscale(3),
+    transforms.ToTensor(),
+    transforms.Normalize((0.1307,0.1307,0.1307), (0.3081,0.3081,0.3081))
+])
+train_data = datasets.MNIST(
+    root=".",
+    train=True,
+    transform=data_transform
+)
+data_loader = DataLoader(train_data)
+# (Optional) Something else, like training ...
+trainer = Trainer()
+trainer.fit(model, data_loader)
```

Comment:
Same here.

Reply:
Do you mean that we don't need fine-tuning before quantization in this doc?

Reply:
As it is a tutorial, maybe we still need to make sure the model is well trained. So I suggest using a pre-trained model on MNIST(if you can find one) directly instead of calling 'fit' to do any training-related work.

 Otherwise, train a small model. But is Mobilenet suitable here for training on MNIST and training for only on epoch? If using MNIST, maybe refer to https://github.com/pytorch/examples/blob/master/mnist/main.py ?

Do we need to show the accuracy diff? @yangw1234